src/edit/Toolbar.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-toolbar
+    height="72"
+    color="transparent"
+    class="elevation-0">
+    <v-toolbar-title class="pl-1">Api Video</v-toolbar-title>
+    <v-toolbar-items class="mx-auto">
+      <upload-btn
+        v-if="!fileName"
+        @change="upload"
+        label="Upload Api video"
+        accept="video/mp4"
+        class="upload-btn" />
+      <template v-else>
+        <v-text-field
+          :value="fileName"
+          readonly hide-details filled />
+      </template>
+    </v-toolbar-items>
+  </v-toolbar>
+</template>
+
+<script>
+import get from 'lodash/get';
+import UploadBtn from './UploadBtn.vue';
+
+const MP4_MIME_TYPE = 'video/mp4';
+const FORMAT_ERROR = 'MP4 format is required.';
+
+export default {
+  name: 'tce-api-video-toolbar',
+  inject: ['$elementBus'],
+  props: {
+    element: { type: Object, required: true }
+  },
+  computed: {
+    fileName: ({ element }) => get(element, 'data.fileName', '')
```

Comment:
Think we could replace `get` with optional chaining. 🙃 
```suggestion
    fileName: ({ element }) => element.data?.fileName,
```